packages/lib/bookingSuccessRedirect.ts

```diff
@@ -36,9 +36,31 @@ export const getBookingRedirectExtraParams = (booking: SuccessRedirectBookingTyp
     "location",
   ];
 
-  return (Object.keys(booking) as BookingResponseKey[])
+  const basicParams = (Object.keys(booking) as BookingResponseKey[])
     .filter((key) => redirectQueryParamKeys.includes(key))
     .reduce((obj, key) => ({ ...obj, [key]: booking[key] }), {});
+
+  const serializeResponseValue = (value: any): string => {
+    if (Array.isArray(value)) {
+      return value.join(",");
+    } else if (typeof value === "object" && value !== null) {
+      return encodeURIComponent(JSON.stringify(value));
+    } else {
+      return String(value);
+    }
```

Comment:
"There is still a problem with the location response. You don't get videocallurl for the dynamic video app in the location, so you’re seeing something like `integration_.....`

Reply:
Can you paste the output redirected link and the response of this api : `https://cal.com/api/book/event`

Reply:
Exactly what I said in the earlier comment, currently the params that are passed are taken from have responses object (from the api call : `https://cal.com/api/book/event` )as below
```
"responses": {
    "name": "Magnus",
    "Alias": "jadshiv",
    "email": "shivamjadhav9333@gmail.com",
    "notes": "Notes typos",
    "title": "Chess",
    "guests": [
      "shivam.jadhav1170378@gmail.com"
    ],
    "location": {
      "value": "integrations:google:meet",
      "optionValue": ""
    }
  }
```

So I am using this as it is , if we want to have the video call url we need to change the existing code, who will give the approval for this ?

Reply:
I have added meeting url in responses params.